codes/quantum/properties/symmetry_protected.yml

```diff
@@ -0,0 +1,31 @@
+code_id: symmetry_protected
+
+name: 'Symmetry-protected self-correcting quantum code'
+introduced: '\cite{arXiv:1805.01474}'
+
+description: |
+  The physical degrees of freedom of the code correspond to a physical lattice with finite-dimensional local Hilbert space. The codespace is the degenerate ground space of a Hamiltonian that consists of terms localized in space. The system is required to have a symmetry not spontaneously broken, with the symmetry-protected self-correcting property described as follows.
+
+  Suppose the physical state is \(\rho(0)\) at time \(t=0\), and is evolved to \(\rho(t)\) by the Hamiltonian plus a small coupling to a thermal bath with a low enough temperature, in the way that is local on the lattice and respects the symmetry. Then the memory time \(\tau_\text{mem}\) is defined to be the maximal \(t\) such that \(\left\|\Phi_\text{ec}(\rho(t))-\rho(0)\right\|_1\leq\epsilon\), where \(\Phi_\text{ec}\) is the error correcting map and \(\epsilon\) a fixed error rate.
```

Comment:
It'll be good to make the inequality into a display equation; see code_id:shor_nine on how to do that.